packages/manager/modules/pci/src/projects/project/data-processing/job-details/job-dashboard/metrics-token/metrics-token.controller.js

```diff
@@ -0,0 +1,18 @@
+import { GRAFANA_URL, GUIDES_MONITOR_JOB_URL } from './metrics-token.constants';
+
+export default class {
+  /* @ngInject */
+  constructor($state) {
+    this.$state = $state;
+    this.grafanaUrl = GRAFANA_URL;
+    this.docsMonitoringUrl = GUIDES_MONITOR_JOB_URL;
+  }
+
+  dismissModal() {
+    this.closeModal();
```

Comment:
This seems unecessary, could the `dismissModal` be replaced by `closeModal` where it's called